arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -0,0 +1,642 @@
+          "description": "The error message that explains the reason value in more detail.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Registry": {
+      "description": "The container registry.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/RegistryProperties",
+          "description": "The container registry properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "RegistryProperties": {
+      "description": "The container registry properties.",
+      "properties": {
+        "loginServer": {
+          "description": "The URL to log into the container registry.",
+          "type": "string",
+          "readOnly": true
+        },
+        "creationDate": {
+          "format": "date-time",
+          "description": "The creation date of the container registry in ISO8601 format.",
+          "type": "string",
+          "readOnly": true
+        },
+        "adminUserEnabled": {
+          "description": "The boolean value that indicates whether admin user is enabled.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "storageAccount": {
+          "$ref": "#/definitions/StorageAccountBase",
+          "description": "The storage account base properties.",
+          "readOnly": true
+        }
+      }
+    },
+    "StorageAccountBase": {
+      "description": "The storage account base properties.",
+      "properties": {
+        "name": {
+          "description": "Storage account name.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RegistryCreateParameters": {
+      "description": "The parameters for creating a container registry.",
+      "required": [
+        "location",
+        "properties"
+      ],
+      "properties": {
+        "location": {
+          "description": "Resource location.",
+          "type": "string"
+        },
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "$ref": "#/definitions/RegistryPropertiesCreateParameters",
+          "description": "The parameters for creating container registry properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "RegistryPropertiesCreateParameters": {
+      "description": "The parameters for creating container registry properties.",
+      "required": [
+        "tenantId",
+        "storageAccount"
+      ],
+      "properties": {
+        "tenantId": {
+          "description": "The tenant id for the current user.",
+          "type": "string"
+        },
+        "adminUserEnabled": {
+          "description": "The boolean value that indicates whether admin user is enabled.",

```

Comment:
Since this is optional. Please specify the default behavior in the description.
"The boolean value that indicates whether admin user is enabled. Default value is false."